catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1644,6 +1685,232 @@ public void setMetacardTypes(List<MetacardType> types) {
         this.metacardTypes = types;
     }
 
+    @Override
+    public CreateResponse create(CreateRequest createRequest) throws IngestException {
```

Comment:
This method has to conform to the CatalogStore interface which only throws IngestException. The resulting transaction call to the endpoint will do the checking and throw a CswException if it doesn't support the typeName which we handle.